projects/packages/waf/src/class-waf-runner.php

```diff
@@ -179,7 +179,7 @@ public static function get_config() {
 			 * @deprecated $next-version$
 			 */
 			// @phan-suppress-next-line PhanDeprecatedClassConstant -- Needed for backwards compatibility.
-			Waf_Rules_Manager::IP_LISTS_ENABLED_OPTION_NAME => get_option( Waf_Rules_Manager::IP_LISTS_ENABLED_OPTION_NAME ),
+			Waf_Rules_Manager::IP_LISTS_ENABLED_OPTION_NAME => get_option( Waf_Rules_Manager::IP_ALLOW_LIST_ENABLED_OPTION_NAME ) || get_option( Waf_Rules_Manager::IP_BLOCK_LIST_ENABLED_OPTION_NAME ),
```

Comment:
Interesting question! After double checking the logic, I don't think these should be changed.

I'm using `get_option` here so that it uses the default value hooks for the block and allow list. This ensures that if the deprecated option was set, and the new options are not, they will base their value on the old one and still return an accurate value :+1:

In the compatibility class, I used `get_raw_option` to bypass those filters, in order to make those default value decisions based on the "true" value of the options in the database.

But at this point, in the runner class, we want those default values to be used :+1: